pkgs/development/libraries/protobuf/generic-v3.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, version, src
```

Comment:
Could it take `version` and `sha256` instead of full `src`? (That would simplify calls to the functions.)